### ISSUE TITLE
Add SQL Agent job management for AlwaysOn scenarios

### ISSUE THREAD

**Issue by thecosmictrickster** [Teutenberg/DBAid#24]
By default, AlwaysOn failovers don't handle SQL Agent jobs. Separate processes need to be implemented to prevent SQL Agent jobs running on secondary replicas and ensure they start/keep running on the [new] primary replica. Something along the lines of what's at https://www.sqlrx.com/alwayson-availability-groups-enable-or-disable-jobs-on-failover/ would do.

**Comment by thecosmictrickster** [comment]
Need to consider AlwaysOn scenarios where there is only manual failover. If server is restarted, it logs the same 1480 event but failover doesn't actually occur; so jobs get disabled on the primary anyway. Not a concern if the monitoring job is created in a disabled state, but at the very least it shold be documented.

**Comment by thecosmictrickster** [comment]
Waiting on merge into master.

**Comment by thecosmictrickster** [comment]
Complete.
